lib/lib.js

```diff
@@ -74,7 +74,12 @@ function maybeTestForPlusOne(config, comment) {
   return /\+1/.test(comment);
 }
 
+exports.formatReviewedByMetaData = formatReviewedByMetaData;
 function formatReviewedByMetaData(reviewer) {
+  if (!reviewer.email) {
```

Comment:
cool, thanks for the explanation!

I think we are good to go now, I confused travis a bit, as I accidentally deleted the branch in between, but now everything is green -- what do you think?